openedx/adg/lms/static/components/student_account/PasswordResetConfirmation.jsx

```diff
@@ -41,6 +44,8 @@ class PasswordResetConfirmation extends React.Component {
       validationMessage: '',
       passwordType1: 'password',
       passwordType2: 'password',
+      eyeIconClasses1: this.hidePasswordClasses,
+      eyeIconClasses2: this.hidePasswordClasses,
```

Comment:
please specify hide in keys as well.

Reply:
This can be show password classes as well. Can't change that.